lib/typescript/class-names-to-type-definition.ts

```diff
@@ -41,26 +49,31 @@ const isValidName = (className: ClassName) => {
 };
 
 export const classNamesToTypeDefinitions = (
-  classNames: ClassNames,
-  exportType: ExportType,
-  quoteType: QuoteType = "single"
+  options: ITypeDefinitionOptions
 ): string | null => {
-  if (classNames.length) {
+  if (options.classNames.length) {
     let typeDefinitions;
 
-    switch (exportType) {
+    const {
+      exportTypeName: ClassNames = "ClassNames",
```

Comment:
Do the default values need to be set here? They should already be defaulted by `cli.ts` if I recall correctly? If not, it would be great to remove this.

If they are needed, we should refer to a constant so these values don't get out of sync.

Reply:
Interesting thing about this—we are actually _not_ using the cli, and I added defaults in case we aren't the only ones using main in a node script. I have another PR planned that would add an officially documented node api.

Reply:
Gotcha, in that case as long as we pull this out into a constant that's shared with the CLI I'm fine with defaulting the value here as well.

Reply:
Should now be using constants. Thanks.